contracts/ROSCA.sol

```diff
@@ -600,15 +606,18 @@ contract ROSCA {
    * this sends all the funds to the foreperson by selfdestructing this contract.
    */
   function emergencyWithdrawal() onlyFromForeperson onlyIfEscapeHatchActive external {
-    LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+	  LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+    bool failedSendingFunds = false;
     // Send everything, including potential fees, to foreperson to disperse offline to participants.
     bool isEthRosca = (tokenContract == address(0));
     if (!isEthRosca) {
       uint256 balance = tokenContract.balanceOf(address(this));
-      // we don't care much about the success of transfer` here as there's not much we can do.
-      tokenContract.transfer(foreperson, balance);
+	    failedSendingFunds = tokenContract.transfer(foreperson, balance);
```

Comment:
the name `failedSendingFunds` is misleading

Reply:
done